### ISSUE TITLE
Scoped mode

### ISSUE THREAD

**Issue by bergie** [noflo/noflo-runtime-msgflo#68]
NoFlo 0.7 added support for IP Objects that can do flow isolation via `scope`: noflo/noflo#373 noflo/noflo#162

We should add an option to the MsgFlo runtime that would isolate and identify message-based execution using the scope instead of groups.

This should be an option so that users can switch to scopes with graphs that are fully utilizing the new Process API.

**Comment by jonnor** [comment]
Why should it be an option, shouldn't we just use this always for NoFlo 0.7 ?

**Comment by bergie** [comment]
Yeah, I realized we may not need a mode for this. With NoFlo 0.7, always post the IP with `scope`, but also set brackets.

Then at the outport side see whether you still get it from `scope`, otherwise fall back to topmost bracket.
